### ISSUE TITLE
Slow bitwise operators

### ISSUE THREAD

**Issue by serge-sans-paille** [serge-sans-paille/pythran#569]
Expressions like

```
#pythran export foo(uint8[], uint8)
def foo(a, b):
    return a & b == a
```

are slow in pythran if ``a.dtype is np.uint8``. Even with uint64 btw. What's wrong?

**Comment by LuisBL** [comment]
arggg, I've do many bitwise operations on uint8.
Say me if there is something I can do to help.

**Comment by LuisBL** [comment]
Did you have a chance to improve speed for np.uint8 & np.uint8 ?

**Comment by serge-sans-paille** [comment]
On Sat, Jan 07, 2017 at 07:48:24AM -0800, Luis wrote:
> Did you have a chance to improve speed for np.uint8 & np.uint8 ?

No, but thanks to your reminder, that will be my next task! Keep in
touch :-)

**Comment by serge-sans-paille** [comment]
@LuisBL just to be sure : any reason why not nusing a uint64 ?

**Comment by LuisBL** [comment]
yes it's because I use `numpy.unpackbits` to store 8 0/1 in only one byte.

Cf. `numpy.bitwise_and` and `numpy.unpackbits`

**Comment by serge-sans-paille** [comment]
I've got the fix *but* SIMD instructions are not used for comparison operators, I'll push it soon.
